tests/src/Kernel/fixtures/markup_rendering_patterns/listing.yml

```
@@ -196,13 +197,14 @@ listing_default_2_col:
   assertions:
     count:
       'div.bcl-listing.bcl-listing--default-2-col': 1
-      'div.row.row-cols-1.row-cols-md-2.g-2': 1
+      'div.row.row-cols-1.row-cols-md-2.g-4': 1
       'div.row': 7
       'div.listing-item': 6
       'img.rounded-1': 5
       'p.card-text.mb-3': 6
       'span.me-4': 6
       'span.mb-2': 12
+      'h5.card-title': 6
```

Add link attributes testing: 
```suggestion
      'h5.card-title': 6
      'a.d-inline-block': 1